app/services/authentication/providers/forem.rb

```diff
@@ -0,0 +1,52 @@
+module Authentication
+  module Providers
+    # GitHub authentication provider, uses omniauth-github as backend
+    class Forem < Provider
+      OFFICIAL_NAME = "Forem".freeze
+      DOMAIN_URL = ApplicationConfig["PASSPORT_OAUTH_URL"] || "https://forem-passport.herokuapp.com".freeze
```

Comment:
The default value should be `passport.forem.com` but I'm currently testing out things with a selfhost instance